### PR TITLE
Update tpm2_policyauthorize.1.md

### PR DESCRIPTION
**Pull request by Nilesh-Deshmukh**
Request you to review the changes.

1. Need to change the hierarchy from null to the owner since tpm2_verifysignature do not create the verification ticket when the hierarchy is null
2.  In  tpm2_create command store the privet and public object in a separate file
3. Typo error while providing the signing key name in tpm2_policyauthorize
4. tpm2_load object is missing before tpm2_unseal command.

### CODE REVIEW

File: man/tpm2_policyauthorize.1.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ verification of the signature on the pcr policy digest using **tpm2_policyauthor
 
 ## Create a TPM object like a sealing object with the authorized policy based authentication:
 * tpm2_createprimary -Q -a o -g sha256 -G rsa -o prim.ctx
-* tpm2_create -Q -g sha256 -u sealing_key.pub -r sealing_key.pub -I- -C prim.ctx\
+* tpm2_create -Q -g sha256 --u sealing_pubkey.pub -r sealing_prikey.pub -I- -C prim.ctx\
```
Comment (williamcroberts) [inline man/tpm2_policyauthorize.1.md]: You have a `--u` now, just `-u`

File: man/tpm2_policyauthorize.1.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ verification of the signature on the pcr policy digest using **tpm2_policyauthor
 ## Create a signing authority
 * openssl genrsa -out signing_key_private.pem 2048
 * openssl rsa -in signing_key_private.pem -out signing_key_public.pem -pubout
-* tpm2_loadexternal -G rsa -a n -u signing_key_public.pem -o signing_key.ctx\
+* tpm2_loadexternal -G rsa -a o -u signing_key_public.pem -o signing_key.ctx\
```
Comment (williamcroberts) [inline man/tpm2_policyauthorize.1.md]: Why does it matter if we don't have a verification ticket?

**Comment by williamcroberts** [comment]
moved: https://github.com/tpm2-software/tpm2-tools/pull/1198

File: man/tpm2_policyauthorize.1.md (hunk 1 above)
Comment (Nilesh-Deshmukh) [reply man/tpm2_policyauthorize.1.md]: I think -u is a correct option to get an output file which contains the public portion of the created object, as per man page.

File: man/tpm2_policyauthorize.1.md (hunk 2 above)
Comment (Nilesh-Deshmukh) [reply man/tpm2_policyauthorize.1.md]: The tpm2_policyauthorize command uses this verification ticket to verify that PCR digest is correctly signed by given public key in unsealing process.
e.g.
tpm2_policyauthorize -S session.ctx -o authorized.policy -f pcr.policy\
-n signing_key.name -t verification.tkt
image

**Pull request closed without merging**